src/Orleans/Configuration/New/Gateway.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading.Tasks;
+using Orleans.Runtime.Configuration;
+
+namespace Orleans.Configuration.New
```

Comment:
Are we sure we are going to call the namespace this? Maybe Orleans.Configuration.vTwo... in case we want to have a v3..